litgpt/config.py

```diff
@@ -1520,6 +1520,27 @@ def norm_class(self) -> Type:
         n_expert=8,
         n_expert_per_token=2,
     ),
+    # https://huggingface.co/mistralai/Mixtral-8x22B-Instruct-v0.1/blob/main/config.json
+    dict(
+        name="Mixtral-8x22B-{}v0.1",
+        hf_config=dict(org="mistralai", name="Mixtral-8x22B-{}v0.1"),
+        padded_vocab_size=32000,
```

Comment:
vocab size is 32768
https://huggingface.co/mistralai/Mixtral-8x22B-Instruct-v0.1/blob/main/config.json#L28